src/modules/land_detector/FixedwingLandDetector.h

```diff
@@ -57,48 +57,46 @@ class FixedwingLandDetector : public LandDetector
 	FixedwingLandDetector();
 
 protected:
-	virtual void _initialize_topics() override;
+	void _initialize_topics() override;
+	void _update_params() override;
+	void _update_topics() override;
+	bool _get_landed_state() override;
+	bool _get_maybe_landed_state() override;
+	bool _get_ground_contact_state() override;
+	bool _get_freefall_state() override;
+	float _get_max_altitude() override;
 
-	virtual void _update_params() override;
-
-	virtual void _update_topics() override;
-
-	virtual bool _get_landed_state() override;
-
-	virtual bool _get_maybe_landed_state() override;
-
-	virtual bool _get_ground_contact_state() override;
```

Comment:
Nitpicks always appreciated! I only recently discovered that the use of final also allows the compiler to devirtualize a lot of calls. I expect to start using it a lot more. In this case for the land detector I've been thinking of using multiple inheritance for VTOL and switch between MC and FW detectors as needed.